nss-day-cohort-52/giffygram-flying-racoons
# Liked posts renders appropriate colored star & like object added to likes array

#### Changes Made
Edited click event on star to set authenticated user to userId in like object sent to API. 
Edited postListItem to find like object that meets criteria and renders appropriate star. 

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout kw-sortfavs2
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.

Load page and press the like button on your post. Like object should be added to likes array and yellow star should be rendered to the page. 

    > What code to look for specifically and what should it be doing?
    [ ] Does app function with change?
    
4. View code file.
    [ ] Confirm file modifications are present as indicated above.
    [ ] Confirm no unused code or extraneous comments exist.
